package.json

```diff
@@ -5,8 +5,7 @@
   "author": "woozyking <runzhou.li@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "start": "node index.js",
-    "dev": "nodemon index.js",
+    "dev": "env $(print-env) nodemon index.js",
```

Comment:
for `print-env`, install it through Python's `pip`. `pip install print-env`, then save `ATLAS_URI=xyz` into a `.env` file in project root